test/functional/lua/utility_functions_spec.lua

```diff
@@ -287,5 +302,13 @@ describe('lua stdlib', function()
   it('vim.pesc', function()
     eq('foo%-bar', exec_lua([[return vim.pesc('foo-bar')]]))
     eq('foo%%%-bar', exec_lua([[return vim.pesc(vim.pesc('foo-bar'))]]))
+
+    local pesc = function(s)
+      return exec_lua('return vim.pesc(...)', s)
```

Comment:
all `vim.shared` functions are available in the tests as well, so just:

    local pesc = helpers.pesc